docs/integrations/bigquery.mdx

```diff
@@ -11,7 +11,16 @@ Supports Bigquery since version 0.3.4
 
 ### BigQuery connection profile
 
-The profiles.yml file to connect to BigQuery should contain a profile in the following format:
+After installing Elementary's dbt package upon running `dbt deps`,
```

Comment:
The cool thing about our migration to Mintlify is that docs are now code. 
You don't need to copy-paste any more. 

Create this part that explains the new profile generation command under the folder called 'snippets'.

Then on each page you want to use it, add this on the top:
`import GenerateElementaryProfile from "../snippets/generate-elementary-profile.mdx";`

And this where you want the actual snippet to be placed:
<GenerateElementaryProfile />

Reply:
Thanks! I've added a custom link for each integration so making a generic snippet isn't as intuitive.
Is there a way to pass props to a snippet so I can do `<GenerateElementaryProfile profile_docs_link={...} />`?